single_chart_migration/migration.sh

```diff
@@ -232,21 +231,26 @@ do_rollback() {
     namespace=$1
     version=$2
 
-    current_px_central_version="2.0.1"
-    if [ `helm list -n px-backup | grep "px-central-2.0.0" | wc -l` -eq 1 ]; then
+    current_px_central_version="2.1.0"
+    if [ `helm list -n px-backup | grep "px-central-2.0.1" | wc -l` -eq 1 ]; then
+        current_px_central_version="2.0.1"
+    elif [ `helm list -n px-backup | grep "px-central-2.0.0" | wc -l` -eq 1 ]; then
         current_px_central_version="2.0.0"
     fi
 
     # job_imagetag
-    if [ $current_px_central_version == "2.0.1" ]; then
-        # TODO: Needs to modify to public tags in 2.0.1 final patch
+    if [ $current_px_central_version == "2.1.0" ]; then
```

Comment:
instead of using else if can we use an array or list of the version that is supported and check for that. 
In long run this will keep adding to the complexity, a better solution would be to say we support all versions ahead of 2.0.0